CONTRIBUTING.md

```diff
@@ -5,8 +5,8 @@ sign off the [Developer Certificate of Origin](https://developercertificate.org/
 
 In order to maintain a high software quality standard, we strongly prefer contributions to follow these rules:
 
-- We pay more attention to the quality of commit messages than most other projects on GitHub. In general, we share the
-  view on how commit message should be written with
+- We pay more attention to the quality of commit messages than most other projects on GitHub do. In general, we share the
```

Comment:
I would put the line break before "In general" instead of mid sentence at "share the"